x/launch/genesis.go

```diff
@@ -23,6 +28,13 @@ func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
 	genesis := types.DefaultGenesis()
 
 	// this line is used by starport scaffolding # genesis/module/export
+	// Get all genesisAccount
+	genesisAccountList := k.GetAllGenesisAccount(ctx)
+	for _, elem := range genesisAccountList {
+		elem := elem
```

Comment:
```suggestion
```

Reply:
Why do we even use pointers in the GenesisState?

Reply:
`GenesisState` structs are protobuf messages and therefore all non-native type fields are pointers

Reply:
So what's the benefit of using a pointer actually and what do you mean by non-native type fields?

Reply:
There is no point it just comes from proto. I actually saw that this is the default generation and we can prevent this by appending the option `[(gogoproto.nullable) = false]`

I will create issues for this good catch